pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -845,6 +846,11 @@ func ValidateManagedNodeGroup(index int, ng *ManagedNodeGroup) error {
 		return err
 	}
 
+	if instanceutils.IsNvidiaInstanceType(SelectInstanceType(ng)) && ng.AMIFamily == NodeImageFamilyBottlerocket {
+		logger.Info("Bottlerocket GPU support is for unmanaged nodegroups only. If your using cli flags pass --managed=false")
```

Comment:
```suggestion
		logger.Info("Bottlerocket GPU support is for unmanaged nodegroups only. If you're using CLI flags pass --managed=false")
```

Reply:
Hah, didn't see your comment. :) Must have been at the same time. :)